test/integration/dockerregistry_imagelayers_test.go

```diff
@@ -0,0 +1,457 @@
+package integration
```

Comment:
add something to the makefile and preferably to travis to see this work?

Reply:
Done.